### PR TITLE
add Fragments example and Readme notes

### PR DESCRIPTION
**Pull request by idkjs**
Added some Fragment examples and notes to the `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -383,6 +384,39 @@ If using directives like `@bsRecord`, `@bsDecoder` or `@bsVariant` in `graphql_p
 
 By default, apollo will add field `__typename` to the queries and will use it to normalize data and manipulate cache (see [normalization](https://www.apollographql.com/docs/react/advanced/caching/#normalization)). This field won't exist on parsed reason objects, since it is not included in the actual query you write, but is added by apollo before sending the query. Since `__typename` is crucial for the cache to work correctly, we need to read data from cache in its raw unparsed format, which is achieved with `readQuery` from `ApolloClient.ReadQuery` defined in `reason-apollo`.
 
+
+## Fragment [:arrow_up:](#table-of-contents)
+
+Using [fragments](https://www.apollographql.com/docs/react/data/fragments/).
+
+An example of cache update could look like this:
+
```
Comment (mbirkegaard) [inline README.md:393]: I don't think you meant to write "cache update"?

**Comment by fakenickels** [comment]
That’s correct @mbirkegaard! We shouldn’t mix yarn.lock with npm locks

**Comment by idkjs** [comment]
Any suggestions on how to fix the `yarn.lock` issue would be appreciated. Thank you.

**Comment by fakenickels** [comment]
@idkjs the package-lock.json should be deleted and only yarn.lock should be kept! If you can update your branch with master  and then only keep your yarn.lock that should do the trick

**Comment by idkjs** [comment]
Looks like that did it.

File: README.md [hunk 2]
```diff
@@ -383,6 +384,39 @@ If using directives like `@bsRecord`, `@bsDecoder` or `@bsVariant` in `graphql_p
 
 By default, apollo will add field `__typename` to the queries and will use it to normalize data and manipulate cache (see [normalization](https://www.apollographql.com/docs/react/advanced/caching/#normalization)). This field won't exist on parsed reason objects, since it is not included in the actual query you write, but is added by apollo before sending the query. Since `__typename` is crucial for the cache to work correctly, we need to read data from cache in its raw unparsed format, which is achieved with `readQuery` from `ApolloClient.ReadQuery` defined in `reason-apollo`.
 
+
+## Fragment [:arrow_up:](#table-of-contents)
+
+Using [fragments](https://www.apollographql.com/docs/react/data/fragments/).
+
+An example of an update could look like this:
```
Comment (mbirkegaard) [inline README.md:392]: This is nitpicky, but I still don't think this line makes sense. There's no update here

```suggestion
Fragments can be defined and use like this:
```

Comment (idkjs) [reply README.md]: Updated to `Fragments can be defined and used like this:`

**Comment by mbirkegaard** [comment]
@all-contributors add @idkjs for docs

**Comment by mbirkegaard** [comment]
@fakenickels Just wanted to try if I could use all-contributors. Would you mind doing it for me?

**Comment by fakenickels** [comment]
@all-contributors add @idkjs for docs

**Comment by fakenickels** [comment]
Hmm looks like it got lost in the moving

**Comment by fakenickels** [comment]
I’m going to set it up again

**Pull request merged**